content/en/docs/getting-started/usage/_index.md

```diff
@@ -8,6 +8,9 @@ no_list: true
 This section describes how to use OpenELB in BGP/ Layer 2 / VIP mode.
 
 ---
+##### **[IP Address Assignment](/docs/getting-started/usage/ip-address-assignment/)**
+
+Describe how openelb assigns IP addresses.
```

Comment:
1. Why the title here is different from the actual title name?
2. `openelb` -> `OpenELB`